config/openshift-serverless-view-eventing-configmaps.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: knative-eventing
+  name: openshift-serverless-view-eventing-configmaps
```

Comment:
where does this come from?

Reply:
why do we need to vendor this, than? we we just want to replace the images?

Reply:
I'm not following, where is the `vendor` involved here?